### DISCUSSION TITLE
install node Artela: "error: RPC failed; curl 92 HTTP/2 stream 5 was not closed cleanly: CANCEL (err 8)"

### DISCUSSION THREAD

**Discussion by binhgithub79** [community/community#122727] [Actions]
### Select Topic Area

Product Feedback

### Body

root@tetest:/home/lulu/go/src# git clone https://github.com/artela-network/artela.git
Cloning into 'artela'...
remote: Enumerating objects: 5989, done.
remote: Counting objects: 100% (813/813), done.
remote: Compressing objects: 100% (439/439), done.
error: RPC failed; curl 92 HTTP/2 stream 5 was not closed cleanly: CANCEL (err 8)
error: 2327 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output
root@tetest:/home/lulu/go/src#

**Comment by CopperEagle** [comment]
Hey @binhgithub79

I tried it for myself and I could not recreate the error. However, the repository is big, over 100 MB. So here are a few things you may want to try:

1.  `git config --global http.version HTTP/1.1`, then clone, then reset using `git config --global http.version HTTP/2`

2.  If you connection is slow then you may need to increase the timeout settings, as in [here](https://dev.to/styt/solving-error-rpc-failed-curl-92-http2-stream-5-was-not-closed-cleanly-cancel-err-8-during-homebrew-installation-4f0f), namely:
``` 
git config --global http.postBuffer 524288000
git config --global http.lowSpeedLimit 0
git config --global http.lowSpeedTime 999999
```

3. [If you](https://stackoverflow.com/questions/33857243/difference-between-clone-in-desktop-and-zip-download) do not need the history of the repository, you can also do a zip download, extract, then call `git init` on it. Fine especially if local use only.

**Reply by binhgithub79** [reply]
thank you very much! I will try it

**Reply by CopperEagle** [reply]
:+1:  If it works, please consider marking it as the answer so others with the same problem can find this as well.

**Reply by binhgithub79** [reply]
root@binhbon:~/artela#artelad init binhbonmoniker
artelad: error while loading shared libraries: libaspect_wasm_instrument.so: cannot open shared object file: No such file or directory
root@binhbon:~/artela#
